pkg/infra/game/message/proposal/condition.go

```diff
@@ -0,0 +1,47 @@
+package proposal
+
+type Attribute uint
+
+const (
+	Health Attribute = iota
+	Stamina
+	TotalAttack
+	TotalDefence
+)
+
+type Comparator uint
+
+const (
+	GreaterThan Comparator = iota
+	LessThan
+)
```

Comment:
```suggestion
const (
	GreaterThan Comparator = iota
	LessThan
	EqualTo
)
```

Do we not want an (admittedly unlikely to be used) equal to comparator, just for completeness?
```suggestion
const (
	GreaterThan Comparator = iota
	LessThan
)
```

Reply:
Unsure, but I think for now we can keep it, and if it's useful for agent teams, we can add it in.